src/grammar.coffee

```diff
@@ -349,6 +351,48 @@ grammar =
     o 'CLASS SimpleAssignable EXTENDS Expression Block', -> new Class $2, $4, $5
   ]
 
+  Import: [
+    o 'IMPORT String',                          -> new Import null, $2
+    o 'IMPORT ImportClause FROM String',        -> new Import $2, $4
+  ]
+
+  ImportClause: [
+    o '{ }',                                    -> new ModuleList [], yes
+    o 'ModuleList OptComma',                    -> new ModuleList $1, no
```

Comment:
This makes `import a, from 'b'` valid (which is invalid JS). (The output is still correct, though.) Do we really want to allow that?